index.html

```diff
@@ -31,6 +31,11 @@ <h2><span class="text-color">I’m a software developer</span>
           of my work and experience! If you like what you see and have a project
           you need coded, don’t hestiate to contact me.
         </p>
+<!--          <p class="supportig-text headline-box">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p> -->
```

Comment:
I think you have some part of the code commented

as it makes your code not clean and lengthy.